src/player/Player.php

```diff
@@ -2526,6 +2530,25 @@ public function getCraftingGrid() : CraftingGrid{
 		return $this->craftingGrid;
 	}
 
+	/**
+	 * Returns the creative inventory shown to the player.
+	 * Unless changed by a plugin, this is usually the same for all players.
+	 */
+	public function getCreativeInventory() : CreativeInventory{
+		return $this->creativeInventory;
+	}
+
+	/**
+	 * To set a custom creative inventory, you need to make a clone of a CreativeInventory instance.
+	 * Make sure to use this method only in or after `PlayerLoginEvent` to sync the contents on the client.
```

Comment:
This line is redundant. There's no event before PlayerLoginEvent where this could be used anyway.